core/domain/stats_domain_test.py

```diff
@@ -673,29 +1023,97 @@ def test_update_learner_action_from_model(self):
         learner_action = stats_domain.LearnerAction('ExplorationStart', {}, 1)
         learner_action_dict = learner_action.to_dict()
 
-        with self.swap(
+        playthrough_id = stats_models.PlaythroughModel.create(
```

Comment:
Same, consider using helper functions to make this simpler.

Reply:
I think the PlaythroughModels are different in each cases. Sometimes it is to test the invalid schema versions

Reply:
Acknowledged.